src/wasm/wasm.cpp

```diff
@@ -796,4 +796,13 @@ void Module::updateMaps() {
   }
 }
 
+uint64_t Table::Segment::constOffset() {
```

Comment:
Let's move this out of wasm.h/.cpp (in general we keep that header minimal and put things in side util files, but also specifically here, see comments below)